src/source/PeerConnection/PeerConnection.c

```diff
@@ -882,6 +899,10 @@ STATUS deinitKvsWebRtc(VOID)
 
     deinitSctpSession();
 
+    srtp_shutdown();
```

Comment:
Nice! How did you find this? Did tooling alert you of a leak?

Reply:
no, the new test I added fails when running the entire test suite. Turns out that its initKvsWebRtc fails at srtp_init.